src/components/buttons/ha-call-api-button.js

```diff
@@ -31,6 +31,7 @@ class HaCallApiButton extends LitElement {
       method: String,
       data: {},
       disabled: Boolean,
+      confirmation: String,
```

Comment:
This is a `js` file with Types in it? 🤔

Reply:
It looks like it, unless I really screwed it up?

Reply:
nah you didn't